readers/api/endpoint_test.go

```diff
@@ -138,7 +145,298 @@ func TestReadAll(t *testing.T) {
 		{
 			desc:   "read page with valid offset and limit",
 			url:    fmt.Sprintf("%s/channels/%s/messages?offset=0&limit=10", ts.URL, chanID),
-			token:  token,
+			token:  fmt.Sprintf("Thing %s", thingToken),
+			status: http.StatusOK,
+			res: pageRes{
+				Total:    uint64(len(messages)),
+				Messages: messages[0:10],
+			},
+		},
+		{
+			desc:   "read page with valid offset and limit",
```

Comment:
It's a good idea to introduce the type of client (thing or user) in the case description.